diff --git a/eza.rb b/eza.rb
index a3fcb20..2d9e644 100644
--- a/eza.rb
+++ b/eza.rb
@@ -14,31 +14,13 @@ class Eza < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "2bdf49feadb1e11bf7a5247cb64cf6a0efc6624ef7af0305f11e53bc0463bf77"
   end
 
-  depends_on "pandoc" => :build
   depends_on "pkgconf" => :build
   depends_on "rust" => :build
-  depends_on "libgit2"
 
   def install
-    ENV["LIBGIT2_NO_VENDOR"] = "1"
+    ENV["LIBGIT2_NO_VENDOR"] = "0"
 
     system "cargo", "install", *std_cargo_args
-
-    bash_completion.install "completions/bash/eza"
-    fish_completion.install "completions/fish/eza.fish"
-    zsh_completion.install  "completions/zsh/_eza"
-
-    args = %w[
-      --standalone
-      --from=markdown
-      --to=man
-    ]
-    system "pandoc", *args, "man/eza.1.md", "-o", "eza.1"
-    system "pandoc", *args, "man/eza_colors.5.md", "-o", "eza_colors.5"
-    system "pandoc", *args, "man/eza_colors-explanation.5.md", "-o", "eza_colors-explanation.5"
-
-    man1.install buildpath.glob("*.1")
-    man5.install buildpath.glob("*.5")
   end
 
   test do
