diff --git a/git.rb b/git.rb
index ae4d5d1..e3b92af 100644
--- a/git.rb
+++ b/git.rb
@@ -22,9 +22,6 @@ class Git < Formula
     sha256 x86_64_linux:   "82fd41404f35e23d7c10efcd9c7dbad28febf07e0b94828c59a96f24aa935d2e"
   end
 
-  depends_on "gettext"
-  depends_on "pcre2"
-
   uses_from_macos "curl", since: :catalina # macOS < 10.15.6 has broken cert path logic
   uses_from_macos "expat"
   uses_from_macos "zlib", since: :high_sierra
@@ -53,11 +50,10 @@ class Git < Formula
     # If these things are installed, tell Git build system not to use them
     ENV["NO_FINK"] = "1"
     ENV["NO_DARWIN_PORTS"] = "1"
-    ENV["PYTHON_PATH"] = which("python3")
+    ENV["PYTHON_PATH"] = which("python")
     ENV["PERL_PATH"] = which("perl")
-    ENV["USE_LIBPCRE2"] = "1"
     ENV["INSTALL_SYMLINKS"] = "1"
-    ENV["LIBPCREDIR"] = Formula["pcre2"].opt_prefix
+    ENV["NO_GETTEXT"] = "1"
     ENV["V"] = "1" # build verbosely
 
     perl_version = Utils.safe_popen_read("perl", "--version")[/v(\d+\.\d+)(?:\.\d+)?/, 1]
