diff --git a/jq.rb b/jq.rb
index f45e82c..63d3a1b 100644
--- a/jq.rb
+++ b/jq.rb
@@ -29,13 +29,12 @@ class Jq < Formula
     depends_on "libtool" => :build
   end
 
-  depends_on "oniguruma"
-
   def install
     system "autoreconf", "--force", "--install", "--verbose" if build.head?
     system "./configure", *std_configure_args,
                           "--disable-silent-rules",
-                          "--disable-maintainer-mode"
+                          "--disable-maintainer-mode",
+                          "--with-oniguruma=builtin"
     system "make", "install"
   end
 
